.github/workflows/node.js.yml

```
@@ -32,16 +32,27 @@ jobs:
               uses: coactions/setup-xvfb@v1
               with:
                   run: npm test
-            - name: Code coverage
+            - name: Toolkit code coverage
               env:
                   # Unset NODE_OPTIONS because of https://github.com/codecov/uploader/issues/475
                   NODE_OPTIONS: ''
               if: ${{ github.repository == 'aws/aws-toolkit-vscode' && ( github.ref == 'master' || github.event_name == 'pull_request' ) }}
               uses: codecov/codecov-action@v4
               with:
-                  flags: macos-unittests
+                  flags: macos-toolkit-unittests
                   verbose: true
-                  file: ./coverage/lcov.info
+                  file: ./coverage/core/lcov.info
+                  token: ${{ secrets.CODECOV_TOKEN }}
+            - name: Amazon Q code coverage
+              env:
+                  # Unset NODE_OPTIONS because of https://github.com/codecov/uploader/issues/475
+                  NODE_OPTIONS: ''
+              if: ${{ github.repository == 'aws/aws-toolkit-vscode' && ( github.ref == 'master' || github.event_name == 'pull_request' ) }}
+              uses: codecov/codecov-action@v4
+              with:
+                  flags: macos-amazonq-unittests
```

i would guess that Q/CW wants this in the `codewhisperer` category so that all of their coverage is under one umbrella. I assume that's what `flags` does
```suggestion
                  flags: codewhisperer
```

---

If I understand codecov correct right now its setup so that amazon q will report all the tests in `macos-amazonq-unittests`, but the `codewhisperer` flag only includes reports that are in `packages/core/src/codewhisperer/` so that you can filter down of a subset of the coverage thats only provided by the codewhisperer module. If we switched the flag to codewhisperer this codecoverage would only report the coverage of `packages/core/src/codewhisperer/`